tests/spectron/popOuts.spectron.js

```diff
@@ -0,0 +1,99 @@
+const Application = require('./spectronSetup');
+const path = require('path');
+
+let app = new Application({});
+
+describe('Tests for pop outs', () => {
+
+    let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
+
+    beforeAll((done) => {
+        return app.startApplication().then((startedApp) => {
+            app = startedApp;
+            done();
+        }).catch((err) => {
+            console.error(`Unable to start application error: ${err}`);
```

Comment:
Could simply be `Unable to start application: ${err}`.

Reply:
agreed